pkg/kubernetes/client/v1alpha1/client.go

```diff
@@ -141,15 +168,46 @@ func InCluster() OptionFunc {
 	}
 }
 
+// WithKubeConfigPath sets kubeconfig path
+// against this client instance
+func WithKubeConfigPath(kubeConfigPath string) OptionFunc {
+	return func(c *Client) {
+		c.KubeConfigPath = kubeConfigPath
+	}
+}
+
+// Clientset returns a new instance of kubernetes clientset
+func (c *Client) Clientset() (*kubernetes.Clientset, error) {
+	config, err := c.Config()
+	if err != nil {
+		return nil, errors.Wrapf(err,
+			"failed to get kubernetes clientset: failed to get kubernetes config: IsInCluster {%t}: kubeConfigPath {%s}",
```

Comment:
can the message use 'KubeConfigPath' instead of 'kubeConfigPath'

Reply:
Updated